forge-game/src/main/java/forge/game/ability/effects/CountersMoveEffect.java

```diff
@@ -147,11 +146,6 @@ public void resolve(SpellAbility sa) {
             Map<CounterType, Integer> countersToAdd = Maps.newHashMap();
 
             for (Card src : srcCards) {
-                // rule 121.5: If the first and second objects are the same object, nothing happens
```

Comment:
Why remove this?

Reply:
code cleanup because removeCounter helper checks for it anyway